server/package.json

```diff
@@ -5,16 +5,16 @@
   "main": "server.js",
   "scripts": {
     "migrate": "pnpm prisma migrate deploy",
-    "compile": "tsc -w",
-    "dev": "node --env-file=.env --watch ./dist/main.js",
+    "compile": "tsc",
+    "dev": "node --env-file=.env ./dist/main.js",
     "start": "tsc && node --env-file=.env ./dist/main.js"
   },
   "author": "Critik-V Core Team",
   "license": "ISC",
   "devDependencies": {
     "@types/cors": "^2.8.17",
     "@types/express": "^4.17.21",
-    "@types/express-session": "^1.17.10",
+    "@types/express-session": "^1.17.10",https://github.com/Critik-V/Critik-V/pull/8/conflict?name=server%252Fpackage.json&ancestor_oid=9dcd47eedf89987f065b9055dc151574a3a78b7d&base_oid=912a543bb32fcbd5df0a54c82b250e283a2d9e7d&head_oid=5efd39374f15e4da3ac2c0f1b98bf7a88ef59cb5
```

Comment:
what is this ? if it still there delete it please !

Reply:
i don't understand